middleware/modelExport.js

```diff
@@ -298,7 +328,18 @@ module.exports = {
         asyncFn.push(helpers.getForm);
         asyncFn.push(helpers.getGrid);
 
-        async.waterfall(asyncFn, function (err, filters, form, grid) {
+        // get the actual export object
```

Comment:
Yeah, okay, I agree. I'll get rid of it.

To use the default Linz export, the least amount code to achieve it is:

```
grid: {
    export: {
        label: 'Export'
    }
}
```

And then override the default export, the least amount of code to achieve it is:

```
grid: {
    export: {
        label: 'Custom export',
        action: 'url/endpoint'
    }
}
```

You can then use any combination of the above, by turn the export object, into an Array.